### PR TITLE
topology: fix wrong parameter for HDMI in sof-jsl-da7219

### PR DESCRIPTION
**Pull request by xiulipan**
Remove the wrong one more 0 in the DAI config for HDMI.

Signed-off-by: Pan Xiuli <xiuli.pan@linux.intel.com>

### CODE REVIEW

File: tools/topology/sof-jsl-da7219.m4 [hunk 1]
```diff
@@ -18,6 +18,8 @@ include(`sof/tokens.m4')
 # Include Jasperlake DSP configuration
 include(`platform/intel/'PLATFORM`.m4')
 
+DEBUG_START
```
Comment (paulstelian97) [inline tools/topology/sof-jsl-da7219.m4:21]: Do we want this on master?

**Comment by xiulipan** [comment]
SOFCI TEST

File: tools/topology/sof-jsl-da7219.m4 (hunk 1 above)
Comment (xiulipan) [reply tools/topology/sof-jsl-da7219.m4:21]: @paulstelian97 This is the shared debug macro for all TPLG.

**Comment by aiChaoSONG** [comment]
SOFCI TEST

**Comment by keqiaozhang** [comment]
verified on JSL chrome.  it can fix the HDMI issue.

**Comment by lgirdwood** [comment]
CI timeout

**Pull request merged**